app/models/action_item.rb

```diff
@@ -0,0 +1,7 @@
+class ActionItem < ApplicationRecord
+    has_many :assignments
+    
+    validates :title, :description, :due_date, presence: true
```

Comment:
due dates should be optional, and probably stored in the assignments model in case the staff wants to change the due date for one participant and not the other